src/features/share-button.js

```diff
@@ -10,7 +10,9 @@ const clickShareButton = ( event ) => {
 	// Do not actually navigate.
 	event.preventDefault();
 
-	const currentUrl = location.href;
+	// Use URL set on the button itself, by default.
+	const currentUrl =
+		button.querySelector( 'a' ).getAttribute( 'href' ) || location.href;
```

Comment:
This works great, however if the link is # it just copies the hash on it's own. Should we expand this so if the link is # it gets the current location?

Also this may be more of an issue because I have altered the default share button to use # in this PR: https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin/pull/85